doc/platforms/x86_64_linux/x86_64_linux.md

```diff
@@ -19,7 +19,7 @@ If it succeeds, you can see the Docker image as follows:
 ```shell
 $ docker images
 REPOSITORY                 TAG                 IMAGE ID            CREATED             SIZE
-edge-orchestration         coconut              502e3c07b01f        3 minutes ago       132MB
+edge-orchestration         coconut             502e3c07b01f        3 minutes ago       132MB
```

Comment:
Please update the image size from 132MB to 185MB.

Reply:
Applied your comment! Thank you for the kind review! ^^